code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -52,7 +52,7 @@
 
 /obj/structure/bed/chair/verb/rotate()
 	set name = "Rotate Chair"
-	set category = "Object"
+	set category = null //So it's only accessible from the right click menu
```

Comment:
Wait why hide this from right click menu anyways
Spamming rotate is the primary use of the verb